src/components/ShareStructure/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default as ShareStructure } from './ShareStructure.vue'
+export { default as EditShareStructure } from './EditShareStructure.vue'
```

Comment:
Why does Edit have to be here? Does the app (or Storybook) need to import it?

Reply:
Added it for the unit testing

Reply:
Storybook is ok without this

Reply:
Weird that unit tests don't import ShareStructure which imports EditShareStructure.